src/main/java/org/codehaus/mojo/versions/DependencyUpdatesReportMojo.java

```diff
@@ -52,8 +55,8 @@
  * @since 1.0-beta-1
  */
 @Mojo( name = "dependency-updates-report",
-       requiresDependencyResolution = ResolutionScope.RUNTIME, threadSafe = true )
-public class DependencyUpdatesReport extends AbstractVersionsReport
+       requiresDependencyResolution = ResolutionScope.RUNTIME, threadSafe = false )
```

Comment:
Why change `threadSafe` true -> false?

Reply:
It was set wrong in the original class to begin with. Multiple threads can share access to the same sink object.

Reply:
new Sink object is create for each plugin executions ...
https://github.com/apache/maven-reporting-impl/blob/maven-reporting-impl-3.2.0/src/main/java/org/apache/maven/reporting/AbstractMavenReport.java#L164

IMHO it is not shred by multiple thread ... maybe I don't see something ...

Reply:
Okay, maybe I was wrong then. 

On the other hand, this needs to be revised. It doesn't fail multithreaded tests without a reason.

Reply:
Updated